Sources/Widgets/Representations/SplineContextRepresentation/index.js

```diff
@@ -154,11 +155,10 @@ function vtkSplineContextRepresentation(publicAPI, model) {
   publicAPI.getSelectedState = (prop, compositeID) => model.state;
 
   function updateAreaVisibility() {
-    model.pipelines.area.actor.setVisibility(model.fill && model.close);
+    model.pipelines.area.actor.setVisibility(model.fill);
```

Comment:
keep logic

Reply:
Isn't it needed to remove that condition if we want the area to be visible regardless of whether the spline is closed or not ?